protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -233,3 +238,42 @@ func buildCertificateDirectories(c *EtcdCluster) []string {
 func notEmpty(v string) bool {
 	return v != ""
 }
+
+// buildEtcdBackupManagerContainer builds a container for the standalone etcd backup manager
+func buildEtcdBackupManagerContainer(c *EtcdCluster) *v1.Container {
+	command := []string{"/etcd-backup"}
```

Comment:
Not suggesting a change in this PR, but in the future it could be useful to let users override the entire command - in case they want or need to use their own backup image.

Reply:
That and set frequency ... Any other ideas @robinpercy? We should probably open an issue to track this stuff.

Reply:
Nothing significant at the moment @chrislovecnm. I suspect we'll want to add some flexibility around the BackupStorage param. But I haven't thought of anything concrete yet.

Should we create a separate issue for these, or add them to the roadmap PR? #4474